mappings/net/minecraft/client/util/DefaultSkinUtils.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS none/bxs net/minecraft/client/util/DefaultSkinUtils
+	FIELD a STEVE Lnone/kp;
+	FIELD b ALEX Lnone/kp;
```

Comment:
`STEVE_SKIN` and `ALEX_SKIN` - imo it's best to name all skin fields consistently.
Similar to naming other textures `_TEX`.